src/wallet/rpcwallet.cpp

```diff
@@ -2575,6 +2577,258 @@ UniValue fundrawtransaction(const JSONRPCRequest& request)
+                            "This command requires that the transaction with the given txid is in the wallet.\n"
+                            "User specifies which output (e.g., a change output) will be used to bump the fee.\n"
+                            "This command will NOT add new inputs.\n"
+                            "Fee must be high enough to pay a new relay fee.\n"
+                            "If tx has child transactions in mempool, the new fee must pay for them as well.\n"
+                            "This command will fail if fee is not high enough or output is not large enough.\n"
+                            "User can specify totalFee, or use RPC setpaytxfee to set a higher fee rate.\n"
+                            "\nArguments:\n"
+                            "1. \"txid\"              (string, required) The txid to be bumped\n"
+                            "2. output index          (numeric, required) The output index to be decremented\n"
+                            "3. options               (object, optional)\n"
+                            "   {\n"
+                            "     \"confTarget\":       \"n\",          (numeric, optional) Confirmation target (in blocks)\n"
+                            "     \"totalFee\":         \"n\",          (numeric, optional) Total fee to pay, in satoshis (not btc)\n"
+                            "     \"maxFee\":           \"n\",          (numeric, optional) Allows user to override bumpfee's max fee\n"
+                            "     \"force\":            \"true/false\", (boolean, optional) Force replacement even if txid is not in user's mempool\n"
+                            "   }\n"
+                            "\nResult:\n"
+                            "{\n"
+                            "  \"txid\":    \"value\",   (string)  The id of the new transaction\n"
+                            "  \"oldfee\":    n,         (numeric) Fee of the replaced transaction\n"
+                            "  \"fee\":       n,         (numeric) Fee of the new transaction\n"
+                            "}\n"
+                            "\nExamples:\n"
+                            "\nBump the fee, get the new transaction\'s txid\n"
+                            + HelpExampleCli("bumpfee", "<txid> <output>")
+                            );
+
+    RPCTypeCheck(request.params, boost::assign::list_of(UniValue::VSTR)(UniValue::VNUM));
+    uint256 hash;
+    hash.SetHex(request.params[0].get_str());
+    std::string strError;
+
+    // retrieve the original tx from the wallet
+    assert(pwalletMain != NULL);
+    LOCK2(cs_main, pwalletMain->cs_wallet);
+    if (!pwalletMain->mapWallet.count(hash))
+        throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, "Invalid or non-wallet transaction id");
+    const CWalletTx& wtx = pwalletMain->mapWallet[hash];
+
+    // check that we are not trying to bump a tx that has already been mined
+    if (wtx.GetDepthInMainChain() > 0)
```

Comment:
You should probably check that the depth is exactly 0, because otherwise the tx is already mined or conflicted with a mined tx and no point in bumping.